nibabel/freesurfer/tests/test_io.py

```diff
@@ -90,6 +91,13 @@ def test_geometry():
         assert any('volume information contained' in str(ww.message) for ww in w)
         assert any('extension code' in str(ww.message) for ww in w)
 
+        # Test reading/writing a surface file in scanner RAS
+        cmd = f"mris_convert --to-scanner {pjoin(data_path, 'surf', 'lh.inflated')} {surf_path}"
```

Comment:
We don't install FreeSurfer to run tests. Could you contribute the converted file to https://codeberg.org/nipy-data/nitest-freesurfer?